modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/utilities/cookies.js

```diff
@@ -19,7 +19,11 @@ class CommerceCookie {
 		return getCookie(this.scope + key, this.cookieType);
 	}
 
-	setValue(key, value, expires, path = '/') {
+	setValue(key, value, expires, path) {
+		if (!path) {
```

Comment:
No actual custom `path` is apparently ever set manually via `setValue` method in Commerce front-end code.

Please evaluate if `path` here should be allowed to be something settable other than `themeDisplay.getPathContext() || '/'`. From the general scope of the PR and the changes you performed elsewhere, it appears to me that cookies' paths are always enforced to be under a `samedomain` policy (`/` or `/<contextPath>`), so there shouldn't be any need to be defensive in this specific case.